### ISSUE TITLE
Confusion between group profiles and related portals

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#1345]
For example, on the KNB there is a group for SASAP members and a portal for the SASAP project: https://knb.ecoinformatics.org/profile/CN=SASAP,DC=dataone,DC=org
https://knb.ecoinformatics.org/portals/SASAP

There may be confusion for our users if they see two similar places in the UI for the SASAP project. Particularly since they both have dataset listings and metrics.

We should consider removing the dataset list and metrics tab from the group profiles. This will reduce the group profiles to just a list of people. This is probably a more appropriate view of the group, since the groups are primarily used for data access control. Portals should be used to collect a list of related datasets.

Thoughts welcome on how this affects our users, particularly since it would be essentially deprecating a feature for our groups (the feature of showing a list of datasets and metrics about those datasets).

**Comment by mbjones** [comment]
@laurenwalker good point.  In the case of SASAP, the CN=SASAP group is the same group that would be contributing to the portal.  So, I would expect the portal to completely replace the profile in this case.  Maybe on the /profile page for the group we can list portals that are associated with that group?  Maybe something similar would be in order for individual users too?

**Comment by laurenwalker** [comment]
Adding this to the Plus/Hosted Repo MVP release, since we need to figure out this overall design before putting the portal editor into production in DataONE.

Things to consider:
- If group profiles are entirely replaced by their corresponding portal, we still need a way to edit that group. (add/remove members, etc.) Should we expose this view in the portal Settings tab? Keep it in the user settings "My groups" view? Both? Will that be confusing?
- How do we find out if a group has a corresponding portal? 
- The same issue stands for user profiles. How do we know that a user has a portal doc that should be shown instead of the generic profile?
- We probably don't want to match up profiles<->portals by matching usernames or group ids to portal labels, since portals could be created with that username/id or id before the user/group has created an LDAP entry
-
